pennylane_qiskit/ibmq.py

```diff
@@ -75,7 +75,7 @@ def login():
                 IBMQ.enable_account(token, **ibmq_kwargs)
 
             active_account = IBMQ.active_account()
-            if active_account is None:
+            if not isinstance(active_account, dict):
```

Comment:
What was the reason for the change here, what can `active_account` now be in this case? Is it boolean, just not necessarily `None`?